packages/medulas-react-components/src/theme/assets/toast/close.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 20 20">
```

Comment:
If these assets are only going to be used with Toast I prefer to move them to `src/context/ToastProvider/assets`.

Reply:
But in this case you will need to change build script to copy those assets to the `lib`

Reply:
oh, you are right. Keep it as you did in first place.